h/static/scripts/karma.config.js

```diff
@@ -57,10 +59,16 @@ module.exports = function(config) {
       extensions: ['.coffee'],
       noParse: [require.resolve('./vendor/katex')],
       configure: function (bundle) {
-        bundle
-          .transform('coffeeify')
-          .plugin('proxyquire-universal');
+        bundle.plugin('proxyquire-universal');
       },
+
+      transform: [
+        'coffeeify',
+        istanbul({
+          ignore: ['**/node_modules/**', '**/katex.js', '**/*.html', '**/*.svg'],
```

Comment:
well, we are currently writing unit tests for annotator right? That is why I left it because it is actually core to what we are doing and we are trying to change some pieces now - before pulling it out. As far as the default ignores, I prefer to be more explicit so the code readers know what we are ignoring.